index.html

```diff
@@ -75,7 +83,7 @@ <h2 class="heading-underline">Learn more</h2>
 								</a>
 							</p>
 						</article>
-						<article class="article">
+						<article class="article element col-md-5 col-sm-12">
```

Comment:
You cannot use column classes and add custom widths as well:

https://github.com/CodeYourFuture/bikes-for-refugees/pull/33/files#diff-3e1a93b11aefa9b36204aaec251b6741R145